packages/test/src/commands/coloredEmbed.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+import type { SlashCommand } from "@lilybird/handlers";
+import { Color, Embed } from "@lilybird/jsx";
+
+export default {
+    post: "GLOBAL",
+    data: { name: "colored_embed", description: "Send colored embed" },
+    run: async (interaction) => {
+        await interaction.deferReply();
+
+        const { ws, rest } = await interaction.client.ping();
+
+        const embed = Embed({ title: "Ping", description: `🏓 WebSocket: \`${ws}ms\` | Rest: \`${rest}ms\``, color: Color.SpringGreen });
```

Comment:
What about renaming this file to `.tsx` and use it as components?

Reply:
Sureee